src/extras/helpers/PolarGridHelper.js

```diff
@@ -0,0 +1,94 @@
+import { LineSegments } from '../../objects/LineSegments';
+import { VertexColors } from '../../constants';
+import { LineBasicMaterial } from '../../materials/LineBasicMaterial';
+import { Float32Attribute } from '../../core/BufferAttribute';
+import { BufferGeometry } from '../../core/BufferGeometry';
+import { Color } from '../../math/Color';
+import { _Math as Math } from '../../math/Math';
+
+/**
+ * @author mrdoob / http://mrdoob.com/
+ * @author Mugen87 / http://github.com/Mugen87
+ * @author Hectate / http://www.github.com/Hectate
+ */
+
+function PolarGridHelper( radius, radials, circles, divisions, color1, color2 ) {
+
+    radius = radius || 10;
+    radials = radials || 16;
+    circles = circles || 8;
+    divisions = divisions || 50;
+    color1 = new THREE.Color( color1 !== undefined ? color1 : 0x444444 );
```

Comment:
@Hectate Can you please remove all references to the `THREE` namespace. The dependencies should be resolved by the import statements at the top...